docs/examples/complete-example.py

```diff
@@ -0,0 +1,45 @@
+import csv
+import datetime
+
+from nupic.frameworks.opf.modelfactory import ModelFactory
+
+import model_params
+
+_NUM_RECORDS = 4000
+_INPUT_FILE_PATH = "gymdata.csv"
+lastSeenValue = None
+
+def createModel():
+  return ModelFactory.create(model_params.MODEL_PARAMS)
+
+def runHotgym():
+  model = createModel()
+  model.enableInference({'predictedField': 'consumption'})
+  with open (_INPUT_FILE_PATH) as fin:
+    reader = csv.reader(fin)
+    headers = reader.next()
+    reader.next()
+    reader.next()
+    for i, record in enumerate(reader, start=1):
```

Comment:
Simplication:

```
for i, record in enumerate(islice(reader, _NUM_RECORDS), start=1)):
```

That would eliminate the `isLast = i == _NUM_RECORDS` business.  You'll also need to `import islice from itertools`.